stable_baselines/common/vec_env/base_vec_env.py

```diff
@@ -150,6 +150,19 @@ def unwrapped(self):
         else:
             return self
 
+    def getattr_depth_check(self, name, already_found):
+        """Check if an attribute reference is being hidden in a recursive call to __getattr__
+
+        :param name (str): name of attribute to check for
+        :param already_found (bool): whether this attribute has already been found in a wrapper
+        :return: (str or None) blocked_venv_class
```

Comment:
either missing comment or it needs a better name
I'm not sure what `blocked_venv_class` means.